microsoft-edge/webview2/releasenotes.md

```diff
@@ -14,13 +14,14 @@ keywords: IWebView2, IWebView2WebView, webview2, webview, win32 apps, win32, edg
 The WebView2 team updates the [WebView2 SDK][NuGetGallery] on a six-week cadence.  Review the following content for up-to-date information on product announcements, additions, modifications, and breaking changes to the APIs.  
 
 > [!NOTE]
-> Ensure you re-compile your app after updating the NuGet package.  The WebView team recommends using the Canary channel when developing using the prerelease packages, and the evergreen runtime when using released packages.  For more information, navigate to [Versioning][Webview2ConceptsVersioningMatchingWebview2RuntimeVersions].  
+> Ensure you re-compile your app after updating the NuGet package.  The WebView team recommends using the Canary channel when developing using the prerelease packages, and the evergreen runtime when using release packages.  For more information, navigate to [Versioning][Webview2ConceptsVersioningMatchingWebview2RuntimeVersions].  
  
-<!--## 1.0.816-prerelease  
+## 1.0.824-prerelease  
```

Comment:
Could you please update the link for `AreBrowserAcceleratorKeysEnabled` to be `Webview2ReferenceWin32Icorewebview2experimentalsettings2ViewWebview210824GetArebrowseracceleratorkeysenabled`? Somehow I couldn't add the comment to line 31.

Reply:
done!